pages/linux/rpcclient.md

```diff
@@ -0,0 +1,36 @@
+# rpcclient
+
+> MS-RPC client tool (part of the samba suite).
+> More information: <https://www.samba.org/samba/docs/current/man-html/rpcclient.1.html>.
+
+- Connect to a host:
+
+`rpcclient --user {{domain}}\{{username}}%{{password}} {{ip}}`
+
+- Connect to a host on a domain without a password:
+
+`rpcclient --user {{username}} --workgroup {{domain}} --no-pass {{ip}}`
+
+- Connect to a host, passing the password hash:
+
+`rpcclient --user {{domain}}\{{username}} --pw-nt-hash {{ip}}`
+
+- Display domain users:
+
+`rpcclient $> enumdomusers`
```

Comment:
Can we wrap this in a call to the `rpcclient` command at all? For example, to execute php code directly you can do `php -r '{{insert_code_here}}'`

Reply:
OK, please see in my last commit ; I added one example with remote shell commands in the command line but I think it's good to keep the examples with the `rpcclient` shell as it is the preferred solution when working with the tool.

Reply:
I see. In other pages though, we  haven't used a shell prefix.

/cc @owenvoke / @mebeim / @agnivade for a 2nd opinion.

Reply:
@sbrl I did it previously with [`evil-winrm.md`](https://raw.githubusercontent.com/tldr-pages/tldr/master/pages/common/evil-winrm.md). Isn't it compliant with your guidelines ?

Reply:
Ah, thanks for that  @dhondta! It's not that it doesn't match the guidelines, it's just that I'm always unsure with subshell examples. If it's  been done before, that's good enough for me :slightly_smiling_face:

Reply:
Even pages like vim has internal commands. It's okay.